server/server.js

```diff
@@ -2,9 +2,8 @@ var express = require("express");
 var cors = require("cors");
 const Items = require("./models/itemSchema");
 const Users = require("./models/userSchema");
-const CONNECTION_STRING =
-  "mongodb+srv://Danielle:8L4oyHRhSAiUaBXe@cluster3.o7ort2o.mongodb.net/?retryWrites=true&w=majority";
-
+const CONNECTION_STRING ="mongodb+srv://Danielle:8L4oyHRhSAiUaBXe@cluster3.o7ort2o.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
should we change the connection string back to the original <user>:<password> one?

Reply:
Yes, will change it.